src/get-started/editor.md

```diff
@@ -41,92 +50,132 @@ that's OK, skip ahead to the [next step: Test drive][].
 
 ## Install VS Code
 
-VS Code is a lightweight editor with complete Flutter
-app execution and debug support.
+[VS Code][] is a code editor to build and debug apps.
+With the Flutter plugin installed, you can compile, deploy, and debug
+Flutter apps.
+
+To install the latest version of VS Code,
+follow Microsoft's instructions for the relevant platform:
+
+- [Install on macOS][]
+- [Install on Windows][]
+- [Install on Linux][]
+
+[VS Code]: https://code.visualstudio.com/
+[Install on macOS]: https://code.visualstudio.com/docs/setup/mac
+[Install on Windows]: https://code.visualstudio.com/docs/setup/windows
+[Install on Linux]: https://code.visualstudio.com/docs/setup/linux
+
+## Install the VS Code Flutter plugin
+
+1. Start **VS Code**.
+
+1. Go to **View** <span aria-label="and then">></span>
+   **Command Palette...**.
+
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>P</kbd>.
+
+1. Type `install`.
+
+1. Select **Extensions: Install Extensions**.
+
+1. Type `flutter` in the extensions search field.
+
+1. Select **Flutter** in the list.
+
+1. Click **Install**.
+   Installing the Flutter plugin also installs the Dart plugin.
 
-* [VS Code][], latest stable version
+## Validate your VS Code setup with the Flutter Doctor
 
-## Install the Flutter and Dart plugins
+1. Go to **View** <span aria-label="and then">></span>
+   **Command Palette...**.
 
- 1. Start VS Code.
- 1. Invoke **View > Command Palette...**.
- 1. Type "install", and select
-    **Extensions: Install Extensions**.
- 1. Type "flutter" in the extensions search field,
-    select **Flutter** in the list, and click **Install**.
-    This also installs the required Dart plugin.
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>P</kbd>.
 
-## Validate your setup with the Flutter Doctor
+1. Type `doctor`.
 
- 1. Invoke **View > Command Palette...**.
- 1. Type "doctor", and select the
-    **Flutter: Run Flutter Doctor**.
- 1. Review the output in the **OUTPUT** pane for any issues.
-    Make sure to select Flutter from the dropdown
-    in the different Output Options.
+1. Select the **Flutter: Run Flutter Doctor**.
+
+1. Review the output in the **OUTPUT** pane for any issues.
+   Make sure to select Flutter from the dropdown
+   in the different Output Options.
 
 </div>
 <div class="tab-pane" id="androidstudio" role="tabpanel" aria-labelledby="androidstudio-tab" markdown="1">
 
-## Install Android Studio
-
-Android Studio offers a complete,
-integrated IDE experience for Flutter.
+## Install Android Studio or IntelliJ IDEA
 
-* [Android Studio][], version 2020.3.1 (Arctic Fox) or later
+Android Studio and IntelliJ IDEA offer complete,
+integrated IDE experience once you install the Flutter plugin.
```

Comment:
I know you didn't write this, but "integrated IDE" sounds weird since the I already means "integrated".